### ISSUE TITLE
using markdown

### ISSUE THREAD

**Issue by ttkethiopia** [team-refcode/refcode.org#52]
## markdown
learnig how to use `markdown`
```
const name= "markdown"
const process= `i am learning about ${name}`
```
![Screenshot (5)](https://user-images.githubusercontent.com/48158451/60671560-56553b80-9e41-11e9-9270-88d458ac427c.png)

**Comment by brentonstrine** [comment]
Great job!!

**Comment by brentonstrine** [comment]
@nai-aung26 I see you closed this ticket. did you mean to do that? I will re-open it since this ticket belongs to @ttkethiopia

**Comment by nai-aung26** [comment]
no, I didn’t mean that. I just want to know what they do. Sorry for the
confusion.

**Comment by brentonstrine** [comment]
No problem! That is why we have test issues!
